Assets/StreamingAssets/Data/Furniture.xml

```diff
@@ -193,6 +192,9 @@
 
     <LocalizationCode>furn_mining_drone_station</LocalizationCode>
     <UnlocalizedDescription>furn_mining_drone_station_desc</UnlocalizedDescription>
+    <RequiredAtmosphere>
+      <Gas hasValue="true" name="O2" />
```

Comment:
The order of the attributes should be name first hasValue second.

Also, we should specify what amount it requires. Having almost cero should be the same as having 0.

And, we shouldn't need to add the ```RequiredAtmosphere``` when the furniture requires no gases.